src/sourmash/commands.py

```diff
@@ -532,14 +542,14 @@ def search(args):
     if args.best_only:
         notify("** reporting only one match because --best-only was set")
 
+    dw = None
```

Comment:
suggest writing this out - `writer` or something. `dw` is pretty minimal :)

Reply:
(and I was confused, to be clear!)

Reply:
renamed!